### PR TITLE
fix: disable riseupvpn experiment

### PR DESCRIPTION
**Pull request by bassosimone**
Fixes: ooni/probe#2001

Proposed Changes

- Comment out  `riseupvpn` from `CircumventionSuite.getTestList`

Screenshot_20220207_155603

### CODE REVIEW

**Review by bassosimone — COMMENTED** [review]
It seems this diff is quite large! Perhaps it depends on another PR and we should be merging that other PR before moving forward with this one?

**Comment by agrabeli** [comment]
Thank you @aanorbel for working on this! 

My only feedback is that I would recommend refraining from removing reference to the RiseupVPN test in the card copy because:

- If we remove reference to RiseupVPN, it will **untranslate** the whole string in all languages
- Our new research fellow will work on fixing the RiseupVPN test and hopefully we will be able to re-enable this test in the app soon

File: app/src/main/java/org/openobservatory/ooniprobe/test/suite/CircumventionSuite.java [hunk 1]
```diff
@@ -35,8 +35,11 @@ public CircumventionSuite(){
                 list.add(new Psiphon());
             if (pm == null || pm.isTestTor())
                 list.add(new Tor());
-            if (pm == null || pm.isTestRiseupVPN())
-                list.add(new RiseupVPN());
+            /* TODO (aanorbel): Riseup VPN Disabled.
+                The riseupvpn experiment has been quite flaky for quite some time.
+                To be enabled only when test is fixed or removed if deemed necessary.
+                if (pm == null || pm.isTestRiseupVPN())
+                list.add(new RiseupVPN());*/
```
Comment (bassosimone) [inline app/src/main/java/org/openobservatory/ooniprobe/test/suite/CircumventionSuite.java:42]: I'd just remove the commented out code: we can always go back and see the git history to see how we removed riseupvpn to learn how to add it back.

**Review by bassosimone — COMMENTED** [review]
Looks good to me!

**Review by agrabeli — COMMENTED** [review]
LGTM. Thank you @aanorbel !

**Pull request merged**